linaro/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+
+RUN apt-get update && apt-get install --yes --no-install-recommends \
+  cmake \
+  wget \
+  xz-utils \
+  && rm --recursive --force /var/lib/apt/lists/*
+
+RUN mkdir -p /opt/linaro-4.9.4 && wget -qO- http://releases.linaro.org/components/toolchain/binaries/latest-4/arm-linux-gnueabihf/gcc-linaro-4.9.4-2017.01-i686_arm-linux-gnueabihf.tar.xz | tar -xJv --strip-components=1 -C /opt/linaro-4.9.4
```

Comment:
I was going to make a comment here about how this line is really long and hard to read... but as I was refactoring it on multiple lines, I started to recognize the elegance of this approach. However... that doesn't change the fact that it is _really_ long. What if you define the download URL in a variable on the line above to shorten this up a bunch?

Reply:
I'd rather not use a permanent environment variable.

Maybe an inline variable?
```
RUN mkdir -p /opt/linaro-4.9.4 \
  && txz="http://releases.linaro.org/components/toolchain/binaries/latest-4/arm-linux-gnueabihf/gcc-linaro-4.9.4-2017.01-i686_arm-linux-gnueabihf.tar.xz" \
  && wget -qO- ${txz} | tar -xJv --strip-components=1 -C /opt/linaro-4.9.4
```

Reply:
I like that a lot more.